src/app/utils/route-utils.ts

```diff
@@ -124,6 +124,20 @@ export abstract class RouteUtils {
     return Object.values(Layer).filter(layer => RouteUtils.ENABLED_LAYERS_FOR_NETWORK[network][layer])
   }
 
+  static getRuntimesForNetwork(network: Network, isEnabled: boolean): Runtime[] {
+    return Object.values(Runtime).filter(
+      runtime => RouteUtils.ENABLED_LAYERS_FOR_NETWORK[network][runtime] === isEnabled,
+    )
+  }
+
+  static getEnabledRuntimesForNetwork(network: Network): Runtime[] {
```

Comment:
This seems too easily confused with getEnabledLayersForNetwork.

I would refactor all these into: `getAllLayersForNetwork(network): {enabled: [], disabled: []}` + ignore consensus where it is consumed.

Reply:
done